app/forms/error_messages.py

```diff
@@ -4,19 +4,19 @@
 # lazy_gettext is used as the string needs to be resolved at runtime
 # for translations to work.
 error_messages = {
-    "MANDATORY_QUESTION": lazy_gettext("Enter an answer to continue."),
-    "MANDATORY_TEXTFIELD": lazy_gettext("Enter an answer to continue."),
-    "MANDATORY_NUMBER": lazy_gettext("Enter an answer to continue."),
-    "MANDATORY_TEXTAREA": lazy_gettext("Enter an answer to continue."),
-    "MANDATORY_RADIO": lazy_gettext("Select an answer to continue."),
-    "MANDATORY_DROPDOWN": lazy_gettext("Select an answer to continue."),
-    "MANDATORY_CHECKBOX": lazy_gettext("Select all that apply to continue."),
-    "MANDATORY_DATE": lazy_gettext("Enter a date to continue."),
-    "MANDATORY_DURATION": lazy_gettext("Enter a duration to continue."),
-    "NUMBER_TOO_SMALL": lazy_gettext("Enter an answer more than or equal to %(min)s."),
-    "NUMBER_TOO_LARGE": lazy_gettext("Enter an answer less than or equal to %(max)s."),
-    "NUMBER_TOO_SMALL_EXCLUSIVE": lazy_gettext("Enter an answer more than %(min)s."),
-    "NUMBER_TOO_LARGE_EXCLUSIVE": lazy_gettext("Enter an answer less than %(max)s."),
```

Comment:
We should update these and the other altered string to use `.format` methods

Reply:
I agree, but I think it should be done in another card. I will raise one.

Reply:
Sounds good to me 👍